Release 1.0

DRSchlaubi opened the pull request.

DRSchlaubi committed:
Update Gradle version

DRSchlaubi committed:
Add Sentry and log level config

DRSchlaubi committed:
Add initial commands

DRSchlaubi committed:
Add Dockerfile
- Run ktlint

DRSchlaubi committed:
Optimize CI

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 040f3f3cdf26cf73514a75ab90d5d9d0081d765c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: cdabd7f60ecaf1737aad9ae2e216d9defb86b65f
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: e84fc30406db688b77a59eedd7f4fd405c45b6a7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 49db0403e6055d801eb79a7f28bfc2e4471910f0
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 88f94a4496f6a35c7ea5761f71d4eb08cbf6514c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/Votebot/bot

---
This message was auto-generated by https://gitcop.com

Niggelgame committed:
feat: add embed builder and migrate current commands to embeds

Niggelgame committed:
fix: change code style to pass style test

Niggelgame committed:
fix: remove comments and use global embed function

Niggelgame committed:
feat: add new branding colors

Niggelgame committed:
feat: add embed builder and migrate current commands to embeds

fix: change code style to pass style test

fix: remove comments and use global embed function

feat: add new branding colors

Niggelgame committed:
Merge pull request #41 from Votebot/feature/embeds

feat: add embed builder and migrate current commands to embeds

NyCodeGHG committed:
Add production error handler

NyCodeGHG committed:
Make whenUsingSentry inline and add contract

DRSchlaubi committed:
Merge pull request #42 from NyCodeGHG/feature/production-error-handler

Add production error handler
